dist/gmusic-theme.js

```diff
@@ -1008,155 +1013,155 @@ Converter.prototype.getValues = function(space) {
 
 module.exports = convert;
 },{"./conversions":3}],5:[function(require,module,exports){
-module.exports = {
```

Comment:
I have no idea, I have my git set up to standardize line endings.  Can't remember how but I did it :laughing: 

I do almost all my dev on windows :+1: